### DISCUSSION TITLE
Transactions on Metamask Mobile is not working?

### DISCUSSION THREAD

**Discussion by ombayoeco11** [wevm/wagmi#2816] [Q&A]
I was wondering if anyone had the same problem like me. I'm creating a dApp, all of my codes is working fine on desktop, the connection to Metamask is working fine (except changing network from dApp i still failed to force change network). But on Mobile, Metamask can connect to my web but whenever i do the transaction here, it doesn't do anything...is there anyone had the same issue as mine?

**Comment by Linch1** [comment]
hey, any news about this? having same issue

**Comment by mpoletiek** [comment]
This has been broken forever. Does anyone have a working version?

**Reply by mpoletiek** [reply]
Currently using the following:

```
"viem": "^2.9.5",
"wagmi": "^2.5.13"
"next": "14.1.4"
```

Next Page Router
